src/Tizen.NUI/src/public/Images/ImageLoading.cs

```diff
@@ -420,7 +420,25 @@ public static Size2D GetClosestImageSize(string filename)
         [EditorBrowsable(EditorBrowsableState.Never)]
         public static Size2D GetOriginalImageSize(string filename)
```

Comment:
How about providing only one API?
GetOriginalImageSize is hidden API, so we can modify it.
'''csharp
public static Size2D GetOriginalImageSize(string filename, bool orientationCorrection = false)
'''

Reply:
I agree